### PR TITLE
Make it easier to call another action from an action

### PR DESCRIPTION
**Pull request by KrauseFx**
Also updated the rocket action to return a :rocket: emoji

### CODE REVIEW

**Comment by marcelofabri** [comment]
This fixes #4371 🎉

**Comment by KrauseFx** [comment]
We had a quick hangout about this, and decided to do the following: 

Instead of being able to call `slack` directly, you'll have to use `OtherAction.slack` (so similar). By prefixing calls with another `fastlane` class it should be easy for us to see dependencies between actions in the future, and will allow us to find newly added dependencies in PRs faster. 

We have a "soft"-rule of not allowing dependencies between actions, unless there is a very good reason for it

**Comment by KrauseFx** [comment]
Appended a commit to make things a bit more complicated for the user 👍

**Comment by KrauseFx** [comment]
Not how I updated the title of the PR 👍

**Comment by mfurtak** [comment]
I also updated the title of the PR 😉 

Looks good to me!

File: fastlane/docs/README.md [hunk 1]
```diff
@@ -79,11 +79,11 @@ From then on, you can just start using your action in your `Fastfile`.
 
 If you think your extension can be used by other developers as well, let me know, and we can bundle it with `fastlane`.
 
-To call another action from within your action, use the following code:
+To call another action from within your action, just use the same code you would use in a `Fastfile`:
 
 ```ruby
-  Actions::DeliverAction.run(text: "Please input your password:", 
-                              key: 123)
+deliver(text: "Please input your password:",
```
Comment (marcelofabri) [inline fastlane/docs/README.md]: shouldn't this be updated?

Comment (KrauseFx) [inline fastlane/docs/README.md]: Very good point, will do that

**Comment by KrauseFx** [comment]
Ready for Review 👍

**Comment by KrauseFx** [comment]
Ready to be merged whenever I get a +1

**Comment by KrauseFx** [comment]
We can try the new GitHub merge and squash featur

**Comment by mfurtak** [comment]
:ship it!

**Comment by KrauseFx** [comment]
Oh yeah, this is now in master 🚀

**Comment by marcelofabri** [comment]
@KrauseFx I think using `other_action` have an impact on the working folder.

For example, I had this to run `cocoapods` action:

```ruby
values = ConfigurationHelper.parse(Fastlane::Actions::CocoapodsAction, {})
Fastlane::Actions::CocoapodsAction.run(values)
```

If I change to `other_action.cocoapods`, the action can't find the Podfile

```
Exit status of command 'pod install' was 1 instead of 0.
[!] No `Podfile' found in the project directory.
```

Is this the expected behavior? Should I open an issue?

**Comment by KrauseFx** [comment]
@marcelofabri Seems like there is just a `Dir.chdir` missing, taking a look at this now 👍

**Comment by KrauseFx** [comment]
I can reproduce using tests, working on a fix

**Comment by KrauseFx** [comment]
Just pushed https://github.com/fastlane/fastlane/pull/4535, nice catch @marcelofabri.

**Pull request merged**